colmena/proxy.py

```diff
@@ -0,0 +1,42 @@
+"""Utilities for interacting with ProxyStore"""
+import proxystore as ps
+
+from typing import Any, Union
+
+to_proxy = ps.to_proxy
+"""Place object in value server and return proxy
+
+Alias of `proxystore.to_proxy() <https://proxystore.readthedocs.io/en/latest/source/proxystore.html#proxystore.to_proxy>`_.
+"""
```

Comment:
I added this alias here so if someone wants to force a value to be passed via the value server, they can use `colmena.proxy.to_proxy()` rather than needing to import `proxystore` as well.

Reply:
I'm not totally sold it is needed, so let's see if we end up using it a lot.

Reply:
That's fair. I use it in the electrolyte design app, but that's only a sample size of 1 :)